packages/compat/src/v1-app.ts

```diff
@@ -188,7 +188,7 @@ export default class V1App implements V1Package {
       'ember-cli-babel': {
         includeExternalHelpers: true,
         compileModules: false,
-        disableDebugTooling: true,
+        disableDebugTooling: false,
```

Comment:
this should fix #235

Reply:
This babel config only handles compiling away non-standard babel plugins in stage1. An addon that doesn't use any non-standard plugins won't end up running this config at all. See https://github.com/embroider-build/embroider/blob/25671a8b43d8938d4c66d3de2fc92acd58563fd6/packages/compat/src/v1-addon.ts#L113-L129

So I don't think this is the right place to compile out the debug tooling. That gets handled in the stage3 babel config. The one you can see in `_babel_config_.js` within the stage2 output.

Reply:
Hmm... I see your point. Since this is coming wholesale from ember-cli-babel should we special case this Babel plug-in and run it later?

Reply:
Also, I’m pretty sure we do run this config, as making this change absolutely corrects the other issue.

Reply:
sorry, I misread. This *is* the stage3 config. 

We need to leave it on here but remove the `this.debugMacrosPlugin()` that was manually adding our own copy. (This probably means we can drop `babel-plugin-debug-macros` from our own dependencies.)

Reply:
@ef4 sounds legit. I'll move this to it's own PR: https://github.com/embroider-build/embroider/pull/249